samigo/samigo-app/src/webapp/jsf/delivery/item/deliverMultipleChoiceSingleCorrect.jsp

```diff
@@ -90,29 +90,14 @@ should be included in file importing DeliveryMessages
 
   </h:panelGroup>
   <script>
-    $('div.mcscFixUp').each(function(index1,elBlockToFix) {
+      var elBlockToFix = $($('script')[$('script').length - 1].parentElement).find('div.mcscFixUp')[0];
```

Comment:
Can you explain this change? You are looking for the script tag's parent and then finding the first div.mcscFixUp? Why is this better than the previous jquery.Each function?

Reply:
see my explanation in [SAK-43625](https://jira.sakaiproject.org/browse/SAK-43625?focusedCommentId=249836&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-249836)

"I believe the root cause of the misalignment was because when there are **more than one** multiple choice questions on the same page, there will be multiple <script> tags written to the page, so the same javascript gets run on all of the multiple choice questions multiple times (throwing off the alignment)...

This fix attempts to fix all of the JIRAs mentioned above by targeting only the current script tag's elements."

Reply:
one unknown though, is that I don't have access to IE, so I couldn't test my patch there.  But it seemed to work ok on OS X with FF and Chrome

Reply:
Searching for a script tag just looks strange .... can you do it on an HTML id or tag instead?

Reply:
I got that from this article

https://stackoverflow.com/questions/403967/how-may-i-reference-the-script-tag-that-loaded-the-currently-executing-script

it mentions that "using an id may cause issues in some browsers"

Reply:
I think that article meant using an id in the script tag.  Did _you_ mean to add an id to some other tag?  e.g. the 'mcscFixUp' tag?  I suppose that's possible.  maybe I can use the question id?   I'll have to look into that.

Reply:
any objection to using a style class as a selector instead of an id?  jsf prepends it's dynamic id stuff to form element ids, and I didn't know of a good way to get jquery to select that dynamic id.  Using a style classname seemed like a good compromise, which I found from the example in https://stackoverflow.com/a/16483521

Reply:
added additional commit 999cbe9da